person.rb

```diff
@@ -1,10 +1,18 @@
 require_relative 'corrector'
+require './rental'
 
 class Person
   attr_accessor :name, :age
   attr_reader :id
 
-  def initialize(age: 11, name: 'Unknown', parent_permission: true); end
+  def initialize(age: 11, name: 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @corrector = Corrector.new
+    @rentals = []
+  end
```

Comment:
- [ ] The `Person` class should have a method for adding rentals. The method should take in two parameters (`book` and `date`) and return a Rental instance that takes in `date`, `book` and, `self` as parameters.

Reply:
- [ ] The issue here has been properly resolved